MANIFEST.in

```
@@ -0,0 +1,2 @@
+recursive-include zppy-interfaces zppy_interfaces/global_time_series/zppy_land_fields.csv
+recursive-include zppy-interfaces zppy_interfaces/global_time_series/index_template.html
```

This command is for recursively including all files of a given type in the package:
```suggestion
recursive-include zppy_interfaces *.csv
recursive-include zppy_interfaces *.html
```
This is not the syntax to use for including individual files, which I would not recommend. You should not have cvs or html files in the python package that you don't want to include in the conda package.

---

@xylar Hmm I'm still getting the same error even with this change.

---

@forsyth2, I see. I think the problem is 2-fold and I hadn't realized that.  I think we have solved the first problem.  Can you verify that, after calling `pip install .`, you can find the place where `zppy-interfaces` is installed and that both the `csv` and `html` file are included there along with the python files?  I think that will be the case with the `MANIFEST.in` as we have it now.

The next problem is that you are looking for files in a location that e3sm_diags would move them to, but this is not how zppy-interfaces works.  You need to use [importlib-resources](https://docs.python.org/3/library/importlib.resources.html) to find the files within the `zppy-interfaces` package.  I will try to make the relevant suggested changes.